devenv.md

```diff
@@ -0,0 +1,258 @@
+# Setup your LeApp development environment
+
+  Currently, LeApp tool is divided into multiple projects. This means that in order to setup
+a development environment of the LeApp project one has to setup everything piece by piece.
+At this point there are basically two choices, either setup minimal viable environment which
+includes [*Snactor library*](#snactor), [*LeApp daemon*](#daemon) and [*LeApp tool*](#leapp)
+which includes Cockpit UI plugin, or setup a full fledged development environment which contains
+all the bits and pieces mentioned in section [*2*](#repositories). The choice of which kind of
+environment should be setup depends on the needs. However, the recommended one is the full fledged
+environment.
+  In order to start the setup procedure, please follow all steps in section [1 - Prerequisites](#prereq) 
+since every step in the following section [*2*](#repositories) depends on it. Then you can proceed and
+set up each required project. This should be again straigt forward process, as it requires following
+each step mentioned in the project section.
+
+  Because we realize that this is quite complicated and rather tedious process, we prepared a [script](#bscript)
+and a set of pre-prepared [images](#vagrant). 
```

Comment:
I'd also expect an Ansible role for this that we can use for both Vagrant and Docker (via Ansible-container) builds.